assets/www/android/platform.js

```diff
@@ -141,6 +141,14 @@ function sharePage() {
 	);
 }
 
+function showAudio() {
+	if (audioPlayer.isAvailable()) {
+		audioPlayer.createMenuArray();
+	}else {
+		chrome.showNotification("audio unavailable for this page");
```

Comment:
IMO this should be merged if @hy-cheng isnt going to be fixing this.  Someone else can then do the l10n.